sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1,3 +1,24 @@
+namespace Azure.Storage
```

Comment:
If this needs to be in `Azure.Storage` it should probably live in the Common package.  If it's only ever going to be used for Blobs then we should keep it here but put it in the `Azure.Storage.Blobs` namespace.  I'd lean toward keeping it in `Azure.Storage.Blobs` so each Storage service is capable of supporting different audiences as needed.

Reply:
Moved to `Azure.Storage.Blobs`.

Reply:
Does it need to update the API listing?  It's still showing in `Azure.Storage`?

Reply:
My bad, fixed in latest revision.